DataCite.js

```diff
@@ -105,9 +105,11 @@ var testCases = [
 				"seeAlso": [],
 				"attachments": [],
 				"title": "Code criminel de l'empereur Charles V vulgairement appellé la Caroline contenant les loix qui sont suivies dans les jurisdictions criminelles de l'Empire et à l'usage des conseils de guerre des troupes suisses.",
-				"url": "http://dx.doi.org/10.12763/ONA1045",
-				"DOI": "10.12763/ONA1045",
 				"date": "1734",
+				"itemID": "https://doi.org/10.12763/ona1045",
+				"abstractNote": "[object Object],[object Object],[object Object]",
```

Comment:
The abstract looks strange because the [CSL JSON from DataCite](http://data.datacite.org/application/vnd.citationstyles.csl+json/10.12763/ona1045) looks like this:
```
  "abstract": [
    {
      "type": "Other",
      "text": "Le code est accompagné de commentaires de F. A. Vogel, qui signe l'épitre dédicatoire"
    },
    {
      "type": "Other",
      "text": "Reliure 18è siècle"
    },
    {
      "type": "Other",
      "text": "Ex-libris manuscrit \"Ex libris Dufour\""
    }
  ],
```